src/DynatraceMetricExporter.ts

```diff
@@ -255,13 +265,26 @@ export class DynatraceMetricExporter implements PushMetricExporter {
 
 		return out;
 	}
+	private serializeUpDownCounter(metric: MetricData) : string[] {
```

Comment:
```suggestion
	private serializeUpDownCounter(metric: MetricData): string[] {
```
probably just forgot to autoformat. :slightly_smiling_face: